src/pathfinding_service/model/channel_view.py

```diff
@@ -23,6 +23,7 @@ def __init__(
         settle_timeout: int,
         deposit: int = 0,
         reveal_timeout: int = DEFAULT_REVEAL_TIMEOUT,
+        balance_update_nonce: int = 0,
```

Comment:
Is this ever set in the constructor, otherwise just set it to 0 by default

Reply:
Isn't that the constructor where it is set to 0 by default? Or what does `: int = 0` mean?